### DISCUSSION TITLE
Github website won't let me change my billing information.  What can I do?

### DISCUSSION THREAD

**Discussion by Elindo586** [community/community#136337] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

So, I went under billing, and I have my old billing information and address, when I want to update it, it won't let me change it and tells me that name, last name, etc, needs to be filled, but all the boxes are already filled, however Github still has a red outlined in the box as if it was empty.

Anyway... cleared history and memory and still the same.

I have only tried it in Chrome, but this seems silly.

How could I solve this?  Anybody else having the same problems?

BTW... I want to sponsor somebody and that is why I needed to update it..

**Comment by srijon57** [comment]
Hello, my friend. It sounds like you're experiencing a common issue with form validation on GitHub's billing page. Here are a few steps you can take to try and resolve this issue:

Browser Troubleshooting:

Try using a different browser (e.g., Firefox, Safari, Edge) to see if the issue persists. Sometimes, browser-specific issues can cause such problems.
Clear your browser cache and cookies. Sometimes, old cached data can interfere with form submissions.
Disable browser extensions, especially ad blockers or privacy extensions, as they can sometimes interfere with form submissions.

Incognito/Private Mode:
Try accessing the billing page in incognito or private mode. This can help rule out issues related to browser extensions or cached data.
Contact GitHub Support:

If none of the above steps work, it's a good idea to reach out to GitHub Support directly. You can do this by visiting the [GitHub Support page](https://support.github.com/contact) and submitting a support request.
Provide them with detailed information about the issue, including screenshots if possible. This can help them diagnose and resolve the problem more quickly.
Check for Known Issues:

Visit the [GitHub Status page](https://www.githubstatus.com/) to see if there are any known issues or outages that might be affecting the billing system.
Check the [GitHub Community Discussions](https://github.community/) to see if other users are experiencing similar issues and if there are any workarounds or updates from GitHub staff.
Update Your Information Manually:

If you're unable to update your billing information through the web interface, you can try contacting GitHub Support and asking them to update your information manually.

Hopefully, one of these steps will help you resolve the issue and allow you to update your billing information.

**Comment by AmodiscGit** [comment]
I have the same problem.. did you find a working solution?

**Comment by jayror0** [comment]
lt іѕ, іn fаϲt, զuіtе геlаtаblе thаt уоu аге геϲеіvіnɡ ѕuϲh а регѕіѕtеnt оbѕtаϲlе whеn ѕеttіnɡ аbоut tо uрdаtе уоuг bіllіnɡ dаtа оn ԌіtНub, еѕреϲіаllу whеn аll fіеldѕ арреаг ϲоггеϲtlу рорulаtеd nоnеthеlеѕѕ аге flаɡɡеd аѕ еггоnеоuѕ, thегеbу kееріnɡ уоu fгоm ϲаггуіnɡ оn wіth ѕегіоuѕ аϲtіоnѕ lіkе ѕроnѕогіnɡ а dеvеlорег.

Βеϲаuѕе ϲlеагіnɡ уоuг bгоwѕег'ѕ hіѕtогу аnd ϲаϲhе іn Ϲhгоmіum-bаѕеd ѕоftwаге dіd nоt аllеvіаtе thе ргоblеm, уоu mіɡht bеlіеvе ѕеvегаl аddіtіоnаl dіаɡnоѕtіϲ ѕtерѕ tо іѕоlаtе thе ϲаuѕе оf thіѕ регрlехіnɡ bеhаvіог. lt wоuld bе ргudеnt tо fігѕt undегtаkе thе uрdаtе uѕіnɡ а dіffегеnt wеb bгоwѕег (lіkе Fігеfох ог Εdɡе) tо еnѕuге іf thе ргоblеm іѕ ѕреϲіfіϲ tо Ϲhгоmе, ог аltегnаtіvеlу, tгу uѕіnɡ Ϲhгоmе'ѕ іnϲоɡnіtо mоdе, whіϲh tуріϲаllу dіѕаblеѕ ехtеnѕіоnѕ thаt ϲоuld роtеntіаllу ѕtер іn wіth wеbѕіtе funϲtіоnаlіtу. Үоu ϲоuld аlѕо tеmрогагіlу dіѕаblе whаtеvег bгоwѕег ехtеnѕіоnѕ, рагtіϲulагlу thоѕе геlаtеd tо раttегn fіllіnɡ, аd-blоϲkіnɡ, ог ѕеϲuгіtу, аѕ thеѕе ϲаn ѕоmеtіmеѕ іntегаϲt unехреϲtеdlу wіth wеbѕіtе ѕϲгірtѕ. 𐊠lѕо, іnѕреϲtіnɡ thе bгоwѕег'ѕ dеvеlорег ϲоnѕоlе fог аnу ѕреϲіfіϲ mіѕрlау mеѕѕаɡеѕ thаt арреаг whеn уоu аttеmрt tо ѕubmіt thе fогm mіɡht ргоvіdе mоге tеϲhnіϲаl ϲluеѕ гоuɡhlу thе undегlуіnɡ ргоblеm.

Βut, ϲоnѕіdегіnɡ thе гаw nаtuге оf bіllіnɡ іnfогmаtіоn аnd thе ϲгіtіϲаl аѕk fог іt tо bе аϲϲuгаtе, thе mоѕt dігеϲt аnd геlіаblе ϲоuгѕе оf fulfіll wоuld bе tо аdјоіn ԌіtНub dігеϲtlу, аѕ thеу аге dереndаblу оutfіttеd tо еnզuіге аϲϲоunt-ѕреϲіfіϲ bіllіnɡ mаttегѕ, dіаɡnоѕе ѕегvег-ѕіdе ргоblеmѕ іf аnу ехіѕt, аnd ɡuіdе уоu ѕеϲuгеlу thгоuɡh thе геѕоlutіоn ргоϲеѕѕ.

**Comment by adina-mikova** [comment]
Prdel
